src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -44,9 +106,4 @@
     -ms-user-select: none; /* Internet Explorer/Edge */
     user-select: none; /* Non-prefixed version, currently supported by Chrome, Opera and Firefox */
   }
-
-  .builder-canvas {
-    width: 9999px;
-    height: 9999px;
```

Comment:
Do we want to leave a standard proportion min-width and min-height here so that initial render on slower machines take up space? (something like 1024x768 could be fine?)

Reply:
this is a good idea, I added a default value here for min-width, min-height

Reply:
done